### PR TITLE
Add spec

### PR DESCRIPTION
**Pull request by norio-nomura**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: objective-c
```
Comment (keplersj) [inline .travis.yml:1]: Shouldn't this be `language: swift`? 😉

**Comment by keplersj** [comment]
I'd love to merge this, but it seems I have lost write access to the repository...

**Comment by Arcanemagus** [comment]
Looks like this repo never had a team associated with it, gave you explicit admin rights and created a team with write access that you are a team maintainer of. If you run into any other issues from the cleanup just ping me and I'll fix it :wink:.

**Comment by norio-nomura** [comment]
Thanks!

**Pull request merged**